File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Products/ProductBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Products;





class ProductBasic implements IProductData
{

    public string $Description;
    public string $Barcode;
    public string $SKU;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public int $ID;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Products/ProductBasic.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Products
{
    public class ProductBasic : IProductData
    {
        public virtual string ID { set; get; }
        public virtual string Code { set; get; }
        public virtual string Name { set; get; }
        public virtual string LanguageCode { set; get; }
        public virtual int Order { set; get; }
        public virtual string Description { set; get; }
        public virtual int State { set; get; }
        public virtual string SKU { set; get; }
        public virtual string Barcode { set; get; }
    }
}
